docs/index.asciidoc

```diff
@@ -49,16 +56,6 @@ In this section, we'll cover:
 You'll retain and expand the functionality of existing stand-alone plugins, but in a more compact, integrated package. 
 In this section, we'll note mapping and behavioral changes, and explain how to preserve current behavior if needed.
 
-[IMPORTANT] 
---
-Uninstall the `logstash-input-snmp` and `logstash-input-snmptrap` plugins _before_ you install the `logstash-integration-snmp` plugin. 
-
-* `bin/logstash-plugin remove logstash-input-snmp` +
-* `bin/logstash-plugin remove logstash-input-snmptrap`
---
-
```

Comment:
No longer a requirement. Removing per review comment in https://github.com/elastic/logstash-docs/pull/1648#discussion_r1596632709.

Reply:
I think it's still necessary. Those plugins that needs to be removed are the individual ones, which are delivered with Logstash by default. I might be wrong, but trying to install the integration plugin with one of those plugins already installed fails, and it requires to remove them. @jsvd

Reply:
The plugin manager checks the integration plugin’s gemspec for the list of plugins it provides and uninstalls those:


```
/tmp/logstash-8.13.2 ❯ bin/logstash-plugin install logstash-integration-snmp     
[....]
[....]
[....]
Installing logstash-integration-snmp
Installation successful
Removing 'logstash-input-snmp' since it is provided by an integration plugin
Resolving dependencies....
Resolving dependencies...
Successfully removed logstash-input-snmp
Removing 'logstash-input-snmptrap' since it is provided by an integration plugin
Successfully removed logstash-input-snmptrap

/tmp/logstash-8.13.2 ❯ bin/logstash-plugin list snmp                        
Using bundled JDK: /tmp/logstash-8.13.2/jdk.app/Contents/Home
logstash-integration-snmp
 ├── logstash-input-snmp
 └── logstash-input-snmptrap
```